.github/workflows/fetch_erddap_data.yaml

```diff
@@ -21,6 +21,15 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+
+    - name: Load secret
+      id: op-load-secret
+      uses: 1password/load-secrets-action@v2
+      with:
+        export-env: false
+      env:
+        OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+        SENTRY_DSN: op://EIMS Administrator/prov-data-sharing-sentry-dsn/credential
```

Comment:
I guess we might use the sentry dsn in another repo and so storing in 1password is worth it?

Reply:
Also just a good POC